website/description_issues.qmd

```diff
@@ -3,20 +3,25 @@ title: "DESCRIPTION file Issues"
 editor: visual
 ---
 
-# Issue 1
+# Formatting Software Names
 
 ## Problem
 
-## Solution
+The automatic spell check has flagged a software name as incorrect and results in a NOTE.
 
-# Issue 2
+## Solution
 
-## Problem
+All references to names of software, packages and API names in the description section of the DESCRIPTION file should be wrapped in single quotes. e.g.: 'ggplot2', 'python', etc.
 
```

Comment:
I just had an idea:
What are your thoughts on starting the ###Details-section here already? 
Then the Solution is a shorter answer to the problem and the example is part of the Details.
I guess it is not really that important but I'm curious on your opinion!

Reply:
I think that is a good change!